libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -130,6 +135,18 @@ export class BotFrameworkAdapter extends BotAdapter {
 
     /**
      * Creates a new BotFrameworkAdapter instance.
+     *
+     * @remarks
+     * Settings for this adapter include:
+     * ```javascript
+     * {
+     *      "appId": "ID assigned to your bot in the [Bot Framework Portal](https://dev.botframework.com/).",
+     *      "appPassword": "Password assigned to your bot in the [Bot Framework Portal](https://dev.botframework.com/).",
+     *      "openIdMetadata": "The Open ID Metadata Endpoint for your bot to use.",
+     *      "oAuthEndpoint": "The OAuth API Endpoint for your bot to use.",
+     *      "channelService": "The optional channel service option for this bot to validate connections from Azure or other channel locations"
+     *  }
```

Comment:
Do we want to document the settings here or just let them follow the link in the docs to BotFrameworkAdapterSettings?

Reply:
I thought it would be useful to have them inline, but I did end up replicating the content in 2 places.